python/lsst/obs/base/ingest_tests.py

```diff
@@ -76,38 +79,31 @@ class IngestTestBase(metaclass=abc.ABCMeta):
     observations).
     """
 
+    instrument = ""
+    """The fully qualified name of the instrument.
+    """
+
+    instrumentName = ""
+    """The name of the instrument.
+    """
+
+    outputRun = "raw"
+    """The name of the output run to use in tests.
+    """
+
     def setUp(self):
         # Use a temporary working directory
         self.root = tempfile.mkdtemp(dir=self.ingestDir)
-        Butler.makeRepo(self.root)
-        self.butler = Butler(self.root, run="raw")
+        createRepo(self.root)
```

Comment:
You probably should add a comment here to say that you are deliberately using the createRepo function here for testing purposes. Maybe we can reassess but I don't think we want to encourage general users to be using the script wrappers in their code.